src/glib/mine/signalproc.h

```diff
@@ -773,6 +773,10 @@ class TAggrResampler {
     TStr RoundStart;
     /// Default value when the buffer is empty
     TFlt DefaultVal;
+    /// Which side of bin interval is closed. The default is ‘left’.
+    TStr Closed;
+    /// Which bin edge label to label bucket with. The default is ‘left’.
```

Comment:
Either `left` or `right`. Resampled time-series time points can correspond to bucket start points (`left`) or bucket end points (`right`).